srcd/dashboards/metadata/welcome.json

```diff
@@ -3,7 +3,7 @@

```

Comment:
Same comment on markdown widget height as from the other dashboard. Keep in mind this metadata one is a bit longer because the content is slightly different.

Another small thing to add to this PR if possible:

1. Rename the progress chart titles:
   1. from `Sync status` to `Metadata collection progress`
   2. from `Collector status` to `Repository collection progress`
2. Change the following markdown line:
   1. From: `You can check the progress at the bottom of this page.`
   2. To: `You can check the progress at the bottom of this page: click **[…]** then _force refresh_ to update.`